packages/kit/src/core/sync/write_types/test/simple-page-server-only/_expected/$types.d.ts

```diff
@@ -20,11 +20,15 @@ export type PageServerLoad<
 	OutputData extends OutputDataShape<PageServerParentData> = OutputDataShape<PageServerParentData>
 > = Kit.ServerLoad<RouteParams, PageServerParentData, OutputData>;
 export type PageServerLoadEvent = Parameters<PageServerLoad>[0];
-export type Errors = null;
+export type ActionData = Kit.AwaitedActions<
+	typeof import('../../../../../../../../+page.server.js').actions
+>;
 export type PageServerData = Kit.AwaitedProperties<
 	Awaited<ReturnType<typeof import('../../../../../../../../+page.server.js').load>>
 >;
 export type PageData = Omit<PageParentData, keyof PageServerData> & PageServerData;
 export type Action = Kit.Action<RouteParams>;
+export type Actions = Kit.Actions<RouteParams>;
+export type ActionEvent = Kit.RequestEvent<RouteParams>;
```

Comment:
Do we need a new type here, given that it's just a `RequestEvent`? Feels like we could just the same `RequestEvent<Params>` we generate for `+server.js` files

Reply:
True, oversight from when ActionEvent still was something more than RequestEvent